### PR TITLE
Add network state to UI

### PR DESCRIPTION
**Pull request by gterzian**
@maciejhirsz I'm running into some issues, and I think it's best to discuss them in here...

### CODE REVIEW

File: packages/backend/src/index.ts [hunk 1]
```diff
@@ -47,3 +50,29 @@ logClients();
...
+http.createServer((request, response) => {
+  const incoming_url = request.url || "";
+  const parsed_url = url.parse(incoming_url, true);
+  const path = parsed_url.path || "";
+  if (path.startsWith("/network_state/")) {
+    const [chain_id, str_node_id] = path.split('/').slice(2);
+    console.log(chain_id, str_node_id);
+    const node_id = Number(str_node_id);
+    const chain = aggregator.getChain(chain_id as Types.ChainLabel);
+    const nodeList = Array.from(chain.nodeList());
```
Comment (gterzian) [inline packages/backend/src/index.ts]: This, for some reason, is empty even after the node has connected to telemetry. Any ideas?

File: packages/frontend/src/components/List/Row.tsx [hunk 2]
```diff
@@ -281,6 +281,13 @@ export class Row extends React.Component<Row.Props, Row.State> {
       setting: 'blocklasttime',
       render: ({ blockTimestamp }) => <Ago when={blockTimestamp} />
     },
+    {
+      label: 'NetworkState',
+      icon: networkIdIcon,
+      width: 100,
+      setting: 'networkstate',
+      render: ({ id }) => React.createElement('a', {href: `/network_state/0/${id}/`, target: "_blank"}, "Network state"),
```
Comment (gterzian) [inline packages/frontend/src/components/List/Row.tsx]: What would be the best way to get the `chainLabel` here?

Comment (maciejhirsz) [reply packages/frontend/src/components/List/Row.tsx]: You could use `getHashData().chain` (from utils), which will pull it from the url. You might want to url encode it (and decode it on the backend).

File: packages/backend/src/index.ts [hunk 3]
```diff
@@ -47,3 +50,29 @@ logClients();
...
+    console.log(nodeList)
+    const node = nodeList.filter((node) => node.id == node_id)[0];
+    console.log(node)
+    let json;
+    if (node) {
+      json = JSON.stringify(node.networkState);
```
Comment (maciejhirsz) [inline packages/backend/src/index.ts]: Isn't `networkState` a JSON string already?

**Comment by gterzian** [comment]
@maciejhirsz Ok, comments addressed!

**Comment by maciejhirsz** [comment]
Cool, will deploy it tomorrow with nginx proxy to the new port.

**Pull request merged**